sigs.yaml

```diff
@@ -1216,7 +1216,7 @@ sigs:
     day: Wednesday
     time: "02:00"
     tz: UTC
-    frequency: monthly - fourth Wednesday every month
+    frequency: monthly - Wednesday, after the fourth Tuesday, every month
```

Comment:
```suggestion
    frequency: monthly - fourth Wednesday every month
```

Reply:
Chatted on slack, change is no longer recommended. Resolving.